Aptoide/aptoide-client-v8
# [feature/MOB-611] Remove Fabric

**What does this PR do?**

   Removes Fabric. Pls check that I haven't forgotten to remove anything.

**Database changed?**

   No

**How should this be manually tested?**

  Check that everything works as expected. Nothing should crash unexpectedly so just use the app normally.

**What are the relevant tickets?**

  [MOB-611](https://aptoide.atlassian.net/browse/MOB-611)

**Code Review Checklist**
